crane-cloud/frontend
# feat: redesign status page and include status series graphs (designs)

# Description

The redesign introduces the following changes:
- The hero section that gives any welcome information to the user
- The overview section with cards of the various Crane Cloud modules
- The time series graphs grouped by parent to avoid having a long scrollable page.

## Type of change


## Trello Ticket ID

https://trello.com/c/N1JDUB2n

## How Can This Been Tested?

First update your `.env` with this `REACT_APP_MONITORING_APP="https://staging-status-api.cranecloud.io/api/v1/statuses"`
and then run this branch locally followed by checking out this route `http://localhost:3000/status` to see the changes.

## Checklist:


# Screenshots

![image](https://github.com/crane-cloud/frontend/assets/63339234/6c953958-d5a4-479e-8929-3100f01086a4)
